conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleBroadcastSuite.scala

```diff
@@ -253,7 +251,7 @@ class MappingRuleBroadcastSuite extends FunSuite with SparkTestBase with LoggerT
     }
   }
 
-  private def cleanupSchema(inputSchemaTree: String): String = {
+  private def cleanupContainsNullProperty(inputSchemaTree: String): String = {
```

Comment:
The mapping rule implementation that uses join creates arrays with `containsNull = false`, while the broadcasting strategy creates arrays that have `containsNull = true` (or it is the opposite - I don't remember). This method just removes this schema property from schema strings so that unit tests ignore that difference.